toolchest_client/tools/tool.py

```diff
@@ -462,7 +462,7 @@ def run(self):
             print(
                 "\nToolchest run failed. "
                 "For support, contact Toolchest with the error log (above) and the following details:\n\n"
-                f"run_id: \n{pretty_print_run_ids}\n"
+                f"run_id: {pretty_print_run_ids}\n"
```

Comment:
The extra line is in there in case there are multiple run IDs (for multiple threads), but if you think it's best, I'm okay with changing this for now since we disabled parallelization

Reply:
It looks very awkward when one run fails – there's a newline, and the one id is partially indented.

Keeping out since it's okay with you.